be/Service/StatisticsChartDataSvc.php

```diff
@@ -43,14 +41,8 @@ public function loadData(array $userNidList): array
      */
     private function loadUsers(array $userNidList): array
     {
-        $userParams = ['deleted' => 0];
-        $usersAmount = \count($userNidList);
-        $limit = self::DEFAULT_USER_AMOUNT;
-
-        if ($usersAmount > 0) {
-            $userParams['userNid'] = $userNidList;
-            $limit = $usersAmount;
-        }
```

Comment:
Tak naprawdę chyba to realizuje cel kejsa, zmiany fryntendowe to tylko reformat i refaktor ??

Reply:
Nie. Z tego co pamiętam, w FE musiałem zmienić miejsce ładowania statystyk.

```
send(LoadData(checkedUsersNids));
```